var/spack/repos/builtin/packages/elfutils/package.py

```diff
@@ -39,9 +39,8 @@ class Elfutils(AutotoolsPackage):
     depends_on('automake', type='build')
     depends_on('autoconf', type='build')
 
-    version('0.163',
-            git='git://git.fedorahosted.org/git/elfutils.git',
-            tag='elfutils-0.163')
+    url      = "https://sourceware.org/elfutils/ftp/0.163/elfutils-0.163.tar.bz2"
```

Comment:
Can you also add:
```python
list_url = "https://sourceware.org/elfutils/ftp"
list_depth = 2
```
This allows `spack versions` and `spack checksum` to pick up newer versions. If you want, you can also add a checksum for the latest release, 0.168.